### PR TITLE
File paths added for sample code in Android native docs

### LINKED ISSUE #5526: Mobile Android: No filepath in some snippets
Missing filepath in:

- Login
- User Profile

As shown in the images:

capture-for-jira-screenshot-20171221-122717-166
capture-for-jira-screenshot-20171221-122755-714
capture-for-jira-screenshot-20171221-123613-570

### PR DESCRIPTION
**Pull request by arshadkazmi42**
Add file paths for sample code of `build.gradle` and `AndroidManifest.xml`

Fix #5526

### CODE REVIEW

File: articles/quickstart/native/android/_includes/_login.md [hunk 1]
```diff
@@ -65,6 +65,8 @@ You do not need to declare a specific `intent-filter` for your activity, because
 The `AndroidManifest.xml` file should look like this:
 
 ```xml
+// app/src/main/AndroidManifest.xml
```
Comment (albertoperdomo) [inline articles/quickstart/native/android/_includes/_login.md]: I don't think `//` comment syntax is valid for XML.
AFAIK, the only way to add a comment in XML is the following:

```
<!-- to be commented xml block goes here -->
```

Can you please change this code block to use the proper syntax or suggest a working alternative?
Thanks!

**Review by albertoperdomo — CHANGES_REQUESTED** [review]
Please change the code block in XML file.
Please my in-line comment for details.

File: articles/quickstart/native/android/_includes/_login.md (hunk 1 above)
Comment (arshadkazmi42) [reply articles/quickstart/native/android/_includes/_login.md]: I was thinking the same, then I saw some comments in same block using `//` so used it. Updating it now

**Comment by arshadkazmi42** [comment]
@albertoperdomo Updated the comment syntax

**Review by albertoperdomo — APPROVED** [review]
Looks good to me! 👍

**Review by albertoperdomo — CHANGES_REQUESTED** [review]
Hello @arshadkazmi42,

I have to request changes again, sorry.

I asked you to change the comment to the right syntax for XML. The syntax you used for Gradle seems to be just fine, according to [Stackoverflow](https://stackoverflow.com/questions/32569883/what-is-the-syntax-for-writing-comments-in-build-gradle-file).

Depending on the language, the syntax will be different, i.e. `//` for Javascript, `#`for Ruby, and so on.

**Comment by arshadkazmi42** [comment]
@albertoperdomo So should I revert back the syntax for gradle file or both the files?

**Comment by albertoperdomo** [comment]
As long as this is the output, how to do is up to you:
* Gradle -> `// ...`
* XML -> `<!-- ... -->`

**Review by albertoperdomo — APPROVED** [review]

**Comment by albertoperdomo** [comment]
Thank you @arshadkazmi42 :)

**Pull request merged**